docs/kqueen.cli_azure_examples.rst

```diff
@@ -0,0 +1,116 @@
+       "user": {
+         "name": "kxxxxxxxxxxxxxxxxxxxxxxxm",
+         "type": "user"
+       }
+     }
+   ]
+
+#. Create new Resource Group
+
+.. code-block:: bash
+
+   $ RESOURCE_GROUP_NAME="kqueen-demo-rg"
+   $ az group create --name "$RESOURCE_GROUP_NAME" --location westeurope
+   {
+     "id": "/subscriptions/8xxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxx2/resourceGroups/kqueen-demo",
+     "location": "westeurope",
+     "managedBy": null,
+     "name": "kqueen-demo",
+     "properties": {
+       "provisioningState": "Succeeded"
+     },
+     "tags": null
+   }
+
+#. Create the Service Principal which will have permissions to manage resources in the specified Subscription
+
+.. code-block:: bash
+
+   $ EMAIL=$(az account list | jq -r '.[].user.name')
+   $ SUBSCRIPTION_ID=$(az account list | jq -r ".[] | select (.user.name == \"$EMAIL\").id")
+   $ SECRET="my_password"
+   $ SERVICE_PRINCIPAL_NAME="kqueen-demo-sp"
+   $ az ad sp create-for-rbac --role="Contributor" --scopes="/subscriptions/$SUBSCRIPTION_ID" --name "$SERVICE_PRINCIPAL_NAME" --password "$SECRET"
+
+#. Get parameters from Azure for KQueen to create provisioner:
+
+.. code-block:: bash
+
+   $ CLIENT_ID="$(az ad sp list --display-name "$SERVICE_PRINCIPAL_NAME" | jq -r '.[].appId')"
+   $ TENANT_ID=$(az ad sp list --display-name "$SERVICE_PRINCIPAL_NAME" | jq -r '.[].additionalProperties.appOwnerTenantId')
+
+
+To create / configure the Azure cluster using KQueen:
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+#. Start KQueen and obtain bearer token:
+
+.. code-block:: bash
+
+   $ git clone https://github.com/Mirantis/kqueen.git
+   $ cd kqueen
+   $ docker-compose -f docker-compose.yml -f docker-compose.demo.yml rm -f # Make sure you are starting from scratch
+   $ docker-compose -f docker-compose.yml -f docker-compose.demo.yml up
+   $ TOKEN=$(curl -s -H "Content-Type: application/json" --data '{ "username": "admin", "password": "default" }' -X POST 127.0.0.1:5000/api/v1/auth | jq -r '.access_token')
+   $ echo $TOKEN
+
+#. Create new organization "testorganization" with new user / password "testusername / testpassword":
+
+.. code-block:: bash
+
+   $ curl -s -H "Authorization: Bearer $TOKEN" -H "Content-Type: application/json" --data '{ "name": "testorganization", "namespace": "testorganization" }' -X POST 127.0.0.1:5000/api/v1/organizations | jq
+   $ ORG_ID=$(curl -s -H "Authorization: Bearer $TOKEN" -H "Content-Type: application/json"  127.0.0.1:5000/api/v1/organizations | jq -r '.[] | select (.name == "testorganization").id')
+   $ echo $ORG_ID
+   $ curl -s -H "Authorization: Bearer $TOKEN" -H "Content-Type: application/json" --data "{ \"username\": \"testusername\", \"organization\": \"Organization:$ORG_ID\", \"role\": \"superadmin\", \"active\": true, \"password\": \"testpassword\" }" -X POST 127.0.0.1:5000/api/v1/users | jq
+
+#. Switch to new user "testusername" and add new Azure Managed Kubernetes Service provisioner:
```

Comment:
Switch to the newly created user and add a new Azure Managed Kubernetes Service provisioner: